include/bluetooth/audio/audio.h

```diff
@@ -33,19 +33,19 @@ extern "C" {
 #define BT_AUDIO_BROADCAST_ID_SIZE               3 /* octets */
 
 /* Audio Context Type, Generic Audio */
-#define BT_AUDIO_CONTEXT_TYPE_PROHIBITED         BIT(0)
-#define BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED        BIT(1)
-#define BT_AUDIO_CONTEXT_TYPE_CONVERSATIONAL     BIT(2)
-#define BT_AUDIO_CONTEXT_TYPE_MEDIA              BIT(3)
-#define BT_AUDIO_CONTEXT_TYPE_GAME               BIT(4)
-#define BT_AUDIO_CONTEXT_TYPE_INSTRUCTIONAL      BIT(5)
-#define BT_AUDIO_CONTEXT_TYPE_VOICE_ASSISTANTS   BIT(6)
-#define BT_AUDIO_CONTEXT_TYPE_LIVE               BIT(7)
-#define BT_AUDIO_CONTEXT_TYPE_SOUND_EFFECTS      BIT(8)
-#define BT_AUDIO_CONTEXT_TYPE_NOTIFICATIONS      BIT(9)
-#define BT_AUDIO_CONTEXT_TYPE_RINGTONE           BIT(10)
-#define BT_AUDIO_CONTEXT_TYPE_ALERTS             BIT(11)
-#define BT_AUDIO_CONTEXT_TYPE_EMERGENCY_ALARM    BIT(12)
+#define BT_AUDIO_CONTEXT_TYPE_PROHIBITED         0
```

Comment:
I would have liked there to be a reference to the source document.

Reply:
done